src/styles/utils/_variables.scss

```
@@ -0,0 +1,2 @@
+$calendar-day-size: 100px;
+$calendar-font-size: 30px;
```

[SASS] - use variables for the main values so that you'll be able to reuse them and give them descriptive names. But don't overuse them, don't create variable for the value that's used just once.
```suggestion
$calendar-font-size: 30px;
```